tests/integration/test_wazuh_db/test_wazuh_db.py

```diff
@@ -137,3 +165,26 @@ def send_chunk_command(command):
     send_chunk_command('global get-agents-by-connection-status 0 active')
     # Check disconnect-agents chunk limit
     send_chunk_command('global disconnect-agents 0 {} syncreq'.format(str(int(time.time()) + 1)))
+
+
+def test_wazuh_db_timeout(configure_sockets_environment,
+                          connect_to_sockets_module,
+                          pre_insert_packages,
+                          pre_set_sync_info):
+
+    """Check that efectively the socket is closed after timeout is reached"""
+
+    def send_row_by_row_command(command):
+        receiver_sockets[0].send(command, size=True)
+        loop = True
+        time.sleep(5)
+        while loop:
+            response = receiver_sockets[0].receive(size=True).decode()
+            status = response.split(" ", 1)[0]
+            if status != 'due':
+                loop = False
+                assert response == 'Socket closed', 'ok'
+                break
```

Comment:
```suggestion
        time.sleep(5)
        while True:
            response = receiver_sockets[0].receive(size=True).decode()
            status = response.split(" ", 1)[0]
            if status != 'due':
                assert response == 'Socket closed', 'ok'
                break
```
With `break` you actually exits the loop, so you can remove the variable.

Reply:
Totally! tanks